.github/workflows/release.yml

```
@@ -160,4 +160,10 @@ jobs:
           files: |
             artifacts/*.*
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: get_version
+        run: |
+          echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Send PR
+        run: |
+          curl --request POST --url 'https://api.github.com/repos/StirlingLabs/XKCP.NET/dispatches' --header 'authorization: token ${{ secrets.DAEMON_HOOKS  }}' --data '{"event_type": "New XKCP Release", "client_payload": {"tag":"${{ env.RELEASE_VERSION }}"}' --header "Accept: application/vnd.github.everest-preview+json"
```

The specific curl I was testing with was:
```suggestion
          curl --request POST --url 'https://api.github.com/repos/StirlingLabs/XKCP.NET/dispatches' --header 'Authorization: TOKEN ${{ secrets.DAEMON_HOOKS  }}' --data '{"event_type": "New XKCP Release", "client_payload": {"tag":"${{ env.RELEASE_VERSION }}"} }' --header "Accept: application/vnd.github.v3+json"
```